code/game/turfs/open.dm

```diff
@@ -86,7 +86,10 @@
 					C.spin(1,1)
 		return 1
 
-/turf/open/proc/MakeSlippery(wet_setting = TURF_WET_WATER) // 1 = Water, 2 = Lube, 3 = Ice
+/turf/open/proc/MakeSlippery(wet_setting = TURF_WET_WATER, min = 0, max = 0) // 1 = Water, 2 = Lube, 3 = Ice
```

Comment:
the comment on this line seems a bit outdated since we use defines for wet_setting now.

Also the min and max argument needs a name change. The names are too generic and could be mistaken for ovject vars, the names should be self-explantory so as to make their function obvious. Finally the min and max arg here do not function like minimum and maximum (demonstrated by the fact that the value of min is always superior to the max in every place the proc is called).